spec/integration/date_time_properties_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
+    class Foo < ActiveFedora::Base
+      property :date, predicate: ::RDF::DC.date do |index|
+        index.type :date
```

Comment:
Date is stored in Solr as a String regardless of the type. When loading from Solr, I think that's the only type information available. Or is there some other way to infer the type? Does the indexer add a date suffix (_dtsi or something) in the field name if it's given a date even if the index.type isn't set? If this is the case then I guess the type could be inferred from the suffix alone.